loops.js

```diff
@@ -51,3 +81,23 @@
 // >> 9
 // ==========================================
 
+// i (variabel)
+//> i (variabel)
+//>> i (variabel)
+
+//Stappenplan
+//1. for loop schrijven;
+//1.1 if statement schrijven;
+//1.2 else if statement schrijven;
+//2. console.log schrijven;
+//2.1 i toevoegen aan console.log;
+//2.2 >i toevoegen aan console.log;
+//2.3 >>i toevoegen aan console.log;
+//3. nodemon loops.js starten
+//4. scripten
+
+for (let i = 0; i < 10; i++) {
+    if ( i < 3) console.log(i);
+    else if ((i > 2 ) && (i < 6)) console.log('>' + i)
+    else if (i > 5 ) console.log('>>' + i)
```

Comment:
Curly bracket bij je if statements is netter en beter leesbaar.
Bij je laatste else hoef je geen else if te schrijven . 
Op regel 101 en 102 de ; achter de console log vergeten  bij regel 100 heb je dat wel.